tools/toitlsp/lsp/doc_handlers.go

```diff
@@ -36,6 +36,11 @@ const (
 	crashReportRateLimit = 30 * time.Second
 )
 
+func isInsideDotPackages(uri lsp.DocumentURI) bool {
+	return strings.Contains(string(uri), "/.packages/") ||
```

Comment:
Maybe only convert to string once?

Reply:
Not really worth it.
The `lsp.DocumentURI` is just a typedef for string, and there isn't actually any work happening.